src/hint_processor/builtin_hint_processor/hint_code.rs

```diff
@@ -432,11 +432,15 @@ pub(crate) const IS_ZERO_NONDET: &str = "memory[ap] = to_felt_or_relocatable(x =
 pub(crate) const IS_ZERO_PACK: &str = r#"from starkware.cairo.common.cairo_secp.secp_utils import SECP_P, pack
 
 x = pack(ids.x, PRIME) % SECP_P"#;
-pub(crate) const IS_ZERO_ASSIGN_SCOPE_VARS: &str = r#"from starkware.cairo.common.cairo_secp.secp_utils import SECP_P
+pub(crate) const IS_ZERO_ASSIGN_SCOPE_VARS_V1: &str = r#"from starkware.cairo.common.cairo_secp.secp_utils import SECP_P
 from starkware.python.math_utils import div_mod
 
 value = x_inv = div_mod(1, x, SECP_P)"#;
 
+pub(crate) const IS_ZERO_ASSIGN_SCOPE_VARS_V2: &str = r#"from starkware.python.math_utils import div_mod
```

Comment:
We're doing the hint code public. It turns out that's useful for users and very unlikely to change.

Reply:
Done!